resources/defaultConfig.json

```diff
@@ -79,5 +79,8 @@
   },
   "simulation_delay": 10000,
   "hardcode_default_provider": "http://127.0.0.1:12345/#c01011611e3501c6b3f6dc4b6d3fe644d21ab301",
-  "token_address": "0xbf9be54Df2001E6Bd044cED0E508d936A9d38b1D"
+  "token_address": "0xbf9be54Df2001E6Bd044cED0E508d936A9d38b1D",
+  "raiden" : {
+    "switch": "OFF"
```

Comment:
@Tunji17 - I recommend making this a boolean, like `"enabled": false`. That way it can be applied in the node like so: `if (this.ctx.config.payments.enabled) { ... }`